### PR TITLE
Load test

### PR DESCRIPTION
**Pull request by lavocatt**
Test was done on this job https://gitlab.com/osbuild/ci/image-builder/-/jobs/1723976612

### CODE REVIEW

**Review by teg — DISMISSED** [review]
Great!

File: test/cases/load_test.sh [hunk 1]
```diff
@@ -33,6 +33,11 @@ export LT_MEAN_RESPONSE_TIME="200"
 export LT_MEDIAN_RESPONSE_TIME="280"
 export LT_PERCENTILE_95_RESPONSE_TIME="500"
 
+export COMPOSE_LT_FAIL_RATIO="1"
+export COMPOSE_LT_MEAN_RESPONSE_TIME="3000"
```
[the remark this answers is not in the document]
Comment (lavocatt) [reply test/cases/load_test.sh:37]: Yeah I don't know either. I did choose those values from experimentation. So, what do you recommend ? Should I remove these mean tests ?

**Review by Gundersanne — APPROVED** [review]
lgtm!

**Review by teg — APPROVED** [review]
looks good to me too! do we know what the issue is with CI?

**Comment by lavocatt** [comment]
It's just a bit complicated to have all the tests passing, I don't know why. It is not related with the modification.

**Pull request merged**